### PR TITLE
Circleci

### PR DESCRIPTION
**Pull request by marderer**

### CODE REVIEW

File: .idea/personal-finance.iml [hunk 1]
```diff
@@ -1,98 +0,0 @@
...
-    <orderEntry type="library" scope="PROVIDED" name="web-console (v3.5.1, rbenv: 2.5.0) [gem]" level="application" />
-    <orderEntry type="library" scope="PROVIDED" name="webpacker (v3.2.2, rbenv: 2.5.0) [gem]" level="application" />
-    <orderEntry type="library" scope="PROVIDED" name="websocket-driver (v0.6.5, rbenv: 2.5.0) [gem]" level="application" />
-    <orderEntry type="library" scope="PROVIDED" name="websocket-extensions (v0.1.3, rbenv: 2.5.0) [gem]" level="application" />
-  </component>
-  <component name="RModuleSettingsStorage">
-    <LOAD_PATH number="0" />
-    <I18N_FOLDERS number="1" string0="$MODULE_DIR$/config/locales" />
-  </component>
-</module>
```
Comment (AleksSenkou) [inline .idea/personal-finance.iml]: @marderer maybe we can ignore them in `.gitignore`?

File: app/channels/application_cable/channel.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 module ApplicationCable
+  # default channel
```
Comment (AleksSenkou) [inline app/channels/application_cable/channel.rb]: @marderer do you like such comments?

Comment (AleksSenkou) [reply app/channels/application_cable/channel.rb]: `ApplicationCable` is default cable, and its channel is default channel. No need in comment

File: app/channels/application_cable/connection.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 module ApplicationCable
+  # default connection
```
Comment (AleksSenkou) [inline app/channels/application_cable/connection.rb]: the same thing is here and in several places more

File: app/channels/application_cable/channel.rb (hunk 2 above)
Comment (marderer) [reply app/channels/application_cable/channel.rb]: @AleksSenkou i dot't know how disable it in reek

Comment (AleksSenkou) [reply app/channels/application_cable/channel.rb]: @marderer can you try to ask help from other students? As I noticed from review - part of guys disabled those comments

Comment (marderer) [reply app/channels/application_cable/channel.rb]: @AleksSenkou Yes, of course

**Pull request merged**